README.md

```
@@ -1,26 +1,175 @@
+3. 개발 리스크 감소
+
+### 릴리즈 노트
+
+- 각 스프린트의 개발 내용을 아카이빙 하기 위해 릴리즈 노트를 만들어 관리했습니다.
+
+| 릴리즈 버전 | 진행기간       | 릴리즈 노트                                                                        | 릴리즈 PR                                                    |
+| ----------- | -------------- | ---------------------------------------------------------------------------------- | ------------------------------------------------------------ |
+| v1.0.0      | 1주차 스프린트 | [1주차 릴리즈 노트](https://github.com/softeer5th/Team2-Getit/releases/tag/v1.0.0) | [v1.0.0](https://github.com/softeer5th/Team2-Getit/pull/27)  |
+| v1.1.0      | 2주차 스프린트 | [2주차 릴리즈 노트](https://github.com/softeer5th/Team2-Getit/releases/tag/v1.1.0) | [v1.1.0](https://github.com/softeer5th/Team2-Getit/pull/76)  |
+| v1.2.0      | 3주차 스프린트 | [3주차 릴리즈 노트](https://github.com/softeer5th/Team2-Getit/releases/tag/v1.2.0) | [v1.2.0](https://github.com/softeer5th/Team2-Getit/pull/141) |
+| v1.3.0      | 4주차 스프린트 | [4주차 릴리즈 노트](https://github.com/softeer5th/Team2-Getit/releases/tag/v1.3.0) | [v1.3.0]()                                                   |
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)